### PR TITLE
Add `All` actions.

### PR DESCRIPTION
**Pull request by take-cheeze**
I don't want the command `docker-compose config --services` run every time so I want the `all` action and the service selection action separate.
Seems create was missing so added.

### CODE REVIEW

**Comment by Silex** [comment]
I'm not sure I'm a fan of this.

How about "all" is the default, and if you use a prefix argument then it asks for the services?

File: docker-compose.el [hunk 1]
```diff
@@ -96,7 +96,7 @@
 
 ;;;###autoload
 (defun docker-compose-build (services args)
-  "Run \"docker-compose build\" using ARGS."
+  "Run \"docker-compose build SERVICES\" using ARGS."
```
Comment (Silex) [inline docker-compose.el]: I prefer `"Run \"docker-compose build ARGS SERVICES\"."`

**Comment by take-cheeze** [comment]
Sometimes `all` is a dangerous operation so I don't think it's good thing to be default.

File: docker-compose.el (hunk 1 above)
Comment (take-cheeze) [reply docker-compose.el]: Yeah right!

**Comment by Silex** [comment]
Good point. Give me a bit of time to think it through.

**Comment by Silex** [comment]
Alright, I think this is the best way. Can you rebase these commits so:

- Squash the "Fix documentation" commits together and reword it to `Improve docker-compose docstrings`.
- Rephrase `Add create.` to `Add docker-compose create`.
- Rephase `Add All actions` to `Add separate action for all services`.

Is that ok for you?

**Comment by take-cheeze** [comment]
I usually don't care about commit messages so you can modify it.
And shouldn't that be in change log instead of commit message?

**Comment by Silex** [comment]
> I usually don't care about commit messages so you can modify it.

I try to follow https://chris.beams.io/posts/git-commit https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html https://github.com/erlang/otp/wiki/writing-good-commit-messages https://github.com/spring-projects/spring-framework/blob/30bce7/CONTRIBUTING.md#format-commit-messages etc :wink: 

> And shouldn't that be in change log instead of commit message?

Both :-)

Anyway, I'll refactor your commits a bit but keep you as the author then merge.

**Comment by Silex** [comment]
Alright merged! Thanks :+1:
